### PR TITLE
get patient with filter

### PR DESCRIPTION
**Pull request by andrt2607**

### CODE REVIEW

**Review by hnfnfl — COMMENTED** [review]
remove transaction

**Comment by andrt2607** [comment]
> remove transaction

done change, check again

File: internal/pkg/handler/patient.go [hunk 1]
```diff
@@ -44,3 +44,16 @@ func (ph *PatientHandler) CreatePatient(ctx *gin.Context) {
 
 	ph.Create(data).Send(ctx)
 }
+
+func (ph *PatientHandler) GetPatient(ctx *gin.Context) {
+	var param dto.ReqParamGetPatient
+	param.IdentityNumber = ctx.Query("identity_number")
+	limit, _ := strconv.Atoi(ctx.Query("limit"))
```
Comment (hnfnfl) [inline internal/pkg/handler/patient.go]: please add error handling

File: internal/pkg/handler/patient.go [hunk 2]
```diff
@@ -44,3 +44,16 @@ func (ph *PatientHandler) CreatePatient(ctx *gin.Context) {
...
+	param.Limit = limit
+	offset, _ := strconv.Atoi(ctx.Query("offset"))
```
Comment (hnfnfl) [inline internal/pkg/handler/patient.go]: please add error handling

**Review by hnfnfl — APPROVED** [review]
LGTM!

**Pull request merged**